### ISSUE TITLE
Project name & name of hash annotation key is potentially misleading

### ISSUE THREAD

**Issue by extracts** [iainc/Markdown-Annotations#3]
Thanks for the effort, I appreciate it.

That said, I did expect something different when I read the project‘s name ("Markdown Annotations") as well as the name required for the hash annotation key ("Annotations"). The ReadMe says:

> The hash annotation key should be “Annotations”, optionally translated into the language of the document.

I thought that an "annotation" is actually a comment or explanatory note about the text in question. By this, it is not directly part of the note itself and clearly (visually or by means of markup) differentiated from it. E.g., [CriticMarkup](https://github.com/CriticMarkup/CriticMarkup-toolkit) has markup to indicate annotations in Markdown text.

As I understood it, your effort is about indicating authorship in Markdown text, which is great. But isn‘t this something different compared to actual "annotations" on a Markdown note?

I work on an app that extracts Markdown "annotation" notes from PDF annotations. Multiple authors annotating the same PDF and also working on the same PDF annotation/Markdown note is not an uncommon use case. So your project looks interesting. But, especially in the context of my app, I wouldn‘t want to (re)use the name "annotation" for indicating authorship as this may be too confusing for users.

**Comment by deletosh** [comment]
Yup, when I saw annotations, my first instinct was WOW! CriticMarkup finally comes to iAW... but....
